src/main/java/com/carhouse/controller/CarSaleController.java

```diff
@@ -161,7 +154,7 @@ public String updateCarSaleSubmit(
     public String deleteCarSale(@PathVariable final Integer carSaleId,
                                 @RequestParam("requestUrl") final String requestUrl) {
         String redirectUrl = requestUrl.replaceAll("\\*\\*\\*", "&")
-                .replaceFirst("http://localhost:[0-9]*", "redirect:");
+                .replaceFirst("http://[a-zA-Z0-9/.]*:[0-9]*", "redirect:");
```

Comment:
Don't understand why it is required at all?
Looks like some problems with redirecting, but I can't understand why it is a problem. Think it's possible to get URL from method annotation or even remove operation might be implemented through AJAX request.

Reply:
I've fixed it